### DISCUSSION TITLE
Only some pages work

### DISCUSSION THREAD

**Discussion by luigi934** [sstaub/NextionX2#8] [Q&A] [answered]
Thank's for your job. I modified my existing working project with arduino mega from original nextion library to yours. I have 21 pages on the nextion, but only the last 3 work (for example pages 21-20-19). If I comment out the functions of a page (for example 19), still only 3 work (18-20-21). What can it depend on? Thank you

Luigi

**Comment by sstaub** [comment, marked as answer]
Try to change list length `#define MAX_LIST_LENGTH         24` 
This is the number of objects you can handle.
